### DISCUSSION TITLE
Will you consider integrating the perplexity API into Copilot as well?

### DISCUSSION THREAD

**Discussion by liuba1223** [logancyang/obsidian-copilot#387] [Ideas]
perplexity can effectively obtain more accurate information. I believe integrating the functionality of perplexity into copilot should lead to unexpected improvements。

**Comment by logancyang** [comment]
It's definitely one option. I will compare a couple of web search providers and make a decision then.

**Comment by tamler** [comment]
I second this. I stumbled on this request while looking for a way to integrate Perplexity into Obsidian. Killer feature.
